### PR TITLE
Upgrade Django to 1.8.

### PR DESCRIPTION
**Pull request by muzaffaryousaf**

### CODE REVIEW

File: submissions/models.py [hunk 1]
```diff
@@ -46,6 +43,12 @@ def __init__(self, *args, **kwargs):
         kwargs['db_index'] = True
         super(AnonymizedUserIDField, self).__init__(*args, **kwargs)
 
+    def deconstruct(self):
+        name, path, args, kwargs = super(AnonymizedUserIDField, self).deconstruct()
+        del kwargs["max_length"]
+        del kwargs["db_index"]
+        return name, path, args, kwargs
+
```
Comment (muhammad-ammar) [inline submissions/models.py]: @dianakhuang Please review the above changes. We are removing `South` for Django 1.8 upgrade, So as per below links alternative of `add_introspection_rules` is `deconstruct `. 

http://stackoverflow.com/questions/29551367/is-there-a-django-1-7-replacement-for-souths-add-introspection-rules
https://docs.djangoproject.com/en/1.8/howto/custom-model-fields/#custom-field-deconstruct-method

Comment (dianakhuang) [inline submissions/models.py]: That looks good to me. :+1:

File: setup.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-submissions',
-    version='0.1.0',
+    version='1.0',
```
Comment (nedbat) [inline setup.py]: This seems like quite a jump in the version number, how did we decide to label it 1.0?

Comment (muhammad-ammar) [inline setup.py]: @nedbat yeah because django 1.8 is a big change.

Comment (muzaffaryousaf) [inline setup.py]: @nedbat Just like Django Big jump :). btw this is just for now, we have to update versions when we do final rebase.

**Comment by edx-webhook** [comment]
Thanks for the pull request, @muzaffaryousaf! I've created [OSPR-2133](https://openedx.atlassian.net/browse/OSPR-2133) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



If you like, you can add yourself to the [AUTHORS](https://github.com/edx/edx-submissions/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by nedbat** [comment]
Can we close this PR?

**Comment by muhammad-ammar** [comment]
@nedbat looks like something happened with bot recently :). I am closing this PR.

**Pull request closed without merging**